openshift/release/create-release-branch.sh

```diff
@@ -18,8 +18,11 @@ git commit -sm ":fire: remove unneeded workflows" .github/
 # Copy the openshift extra files from the OPENSHIFT/main branch.
 git fetch openshift main
 git checkout openshift/main -- .github/workflows openshift OWNERS_ALIASES OWNERS Makefile
-make generate-dockerfiles
-make RELEASE=$release generate-release
+tag=${release/release-/}
+yq write --inplace openshift/project.yaml project.tag "knative-$tag"
+
+make generate-release
```

Comment:
@creydr this is automatically changing the project.tag metadata when creating the branch